ground/gcs/src/plugins/config/expocurve.h

```diff
@@ -0,0 +1,103 @@
+
+/**
+ ******************************************************************************
+ * @file       expocurve.h
+ * @author     Tau Labs, http://taulabs.org, Copyright (C) 2012-2015
+ * @addtogroup GCSPlugins GCS Plugins
+ * @{
+ * @addtogroup ConfigPlugin Config Plugin
+ * @{
+ * @brief Visualize the expo seettings
+ *****************************************************************************/
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ */
+
+#ifndef EXPOCURVE_H
+#define EXPOCURVE_H
+
+
+#include <QWidget>
+
+#include "qwt/src/qwt.h"
+#include "qwt/src/qwt_plot.h"
+#include "qwt/src/qwt_plot_curve.h"
+#include "qwt/src/qwt_scale_draw.h"
+#include "qwt/src/qwt_scale_widget.h"
+#include "qwt/src/qwt_plot_grid.h"
+#include "qwt/src/qwt_legend.h"
+#include "qwt/src/qwt_legend_label.h"
+#include "qwt/src/qwt_plot_marker.h"
+#include "qwt/src/qwt_symbol.h"
+#include <QMutexLocker>
```

Comment:
this has also to be removed: https://github.com/Trex4Git/TauLabs/blob/GCS_Expo_merge/ground/gcs/src/plugins/config/expocurve.h#L97